content/en/continuous_integration/setup_tests/swift.md

```diff
@@ -13,369 +13,407 @@ further_reading:
+
+[1]: https://github.com/DataDog/dd-sdk-swift-testing/releases
+{{% /tab %}}
+{{< /tabs >}}
+
 
 ## Instrumenting your tests
 
 ### Configuring Datadog
 
-To enable testing instrumentation, add the following environment variables to your test target. You must also select your main target in the `Expand variables based on:`  combo or `Target for Variable Expansion` if using test plans.
+To enable testing instrumentation, add the following environment variables to your test target. You must also select your main target in the `Expand variables based on:`  combo or `Target for Variable Expansion` if using test plans:
+
+{{< img src="continuous_integration/swift_env.png" alt="Swift Environments" >}}
 
 For UITests, environment variables need to be set only in the test target, because the framework automatically injects these values to the application.
 
-| Environment variable     | Value                       |
-|--------------------------|-----------------------------|
-| `DD_TEST_RUNNER`           | `true`                         |
-| `DATADOG_CLIENT_TOKEN`     | Your current Datadog client token  |
-| `DD_ENDPOINT`            | The server endpoint of your Datadog Account: `us` (default), `us3`, `eu`, `gov`             |
-| `SRCROOT`                  | `$(SRCROOT)`                  |
+You should set all these variables in your test target:
 
+`DD_TEST_RUNNER`
+: Enables or disables the instrumentation of tests. You should set this value to `$(DD_TEST_RUNNER)` so you can enable and disable test instrumentation with a environment variable defined outside of the test process (for example, in the CI build).<br/>
+**Default**: `false`<br/>
+**Recommended**: `$(DD_TEST_RUNNER)`<br/>
+**Example**: `true`
 
-Optionally, set these environment variables also:
+`DATADOG_CLIENT_TOKEN`
+: The [Datadog Client Token][1] to use to report test results.<br/>
+**Default**: `(empty)`<br/>
+**Example**: `pub0zxxxyyyxxxyyxxxzzxxyyxxxyyy`
 
-| Environment variable     | Value                       |
-|--------------------------|-----------------------------|
-|`DD_ENV `                   | The environment you want to report              |
-|`DD_SERVICE`                | The name of the service you want to report      |
+`DD_SERVICE`
+: Name of the service or library under test.<br/>
+**Default**: `(Repository name)`<br/>
+**Example**: `my-ios-app`
 
+`DD_ENV`
+: Name of the environment where tests are being run. You should set this value to `$(DD_ENV)` so you can use an environment variable at runtime for setting it.<br/>
+**Default**: `none`<br/>
+**Recommended**: `$(DD_ENV)`<br/>
+**Examples**: `ci`, `local`
 
-### Configuring your CI service
+`SRCROOT`
+: This is the path to the project SRCROOT environment variable. The recommended approach is to use `$(SRCROOT)` for the value, as this is automatically set by Xcode.<br/>
```

Comment:
```suggestion
: The path to the project SRCROOT environment variable. Use `$(SRCROOT)` for the value, because it is automatically set by Xcode.<br/>
```